project/build.scala

```diff
@@ -18,13 +19,18 @@ object ContentApiClientBuild extends Build {
     base = file(".")
   )
   .enablePlugins(BuildInfoPlugin)
+  .settings(ScroogeSBT.newSettings: _*)
   .settings(
-      buildInfoKeys := Seq[BuildInfoKey](version),
-      buildInfoPackage := "com.gu.contentapi.buildinfo"
+    buildInfoKeys := Seq[BuildInfoKey](version),
+
+    buildInfoPackage := "com.gu.contentapi.buildinfo",
+    buildInfoObject := "CapiBuildInfo"
   )
   .settings(releaseSettings: _*)
   .settings(sonatypeSettings: _*)
   .settings(
+    unmanagedJars in Compile := (baseDirectory.value / "lib" ** "*.jar").classpath,
```

Comment:
If you get rid of the directory structure inside `/lib` and just whack the jars straight into the lib dir, you can get rid of this line. By default sbt puts `lib/*.jar` onto the classpath.